src/index.ts

```diff
@@ -20,7 +21,9 @@ const jsLogger = (options?: LoggerOptions, destination: string | number = 1): Lo
   }
 
   const pinoOptions: PinoOptions = { ...baseOptions, ...options, transport };
-  return pino(pinoOptions, pino.destination(destination));
+  const logger = pino(pinoOptions, pino.destination(destination));
+  const loggerWithCaller = pinoCaller(logger);
+  return options?.pinoCaller === true ? loggerWithCaller : logger;
```

Comment:
```suggestion
  return options?.pinoCaller === true ? pinoCaller(logger) : logger;
```
i think running less code > clarity in this case.
what do you think?

Reply:
i agree